cuppa/src/main/java/org/forgerock/cuppa/transforms/TagTestBlockFilter.java

```diff
@@ -94,4 +95,18 @@ private boolean shouldRun(Set<String> testTags) {
         intersection.retainAll(b);
         return intersection;
     }
+
+    /**
+     * Tag run state to perform tag based filtering with.
+     */
+    public static final class RunState extends Option<Tags> {
```

Comment:
This name seems a little generic to me. Why did you go for `RunState` over something like `FilteredTagsOption`?